### ISSUE TITLE
little typo in app-editors/pluma-plugins.ebuild

### ISSUE THREAD

**Issue by ghost** [oz123/mate-de-gentoo#56]
```
--- pluma-plugins-1.26.0.ebuild 2021-08-15 18:21:44.167179518 +0200
+++ pluma-plugins-1.26.0-r1.ebuild 2021-08-21 03:03:09.400729502 +0200
@@ -48,7 +48,7 @@
x11-libs/pango[introspection]
x11-libs/gdk-pixbuf:2[introspection]
synctex? ( >=dev-python/dbus-python-0.82 )
- terminal? ( >=xd11-libs/vte-0.52:2.91[introspection] )
+ terminal? ( >=x11-libs/vte-0.52:2.91[introspection] )

)"
```

**Comment by ghost** [comment]
this is my first one - my apologies for any kind of format errors

**Comment by oz123** [comment]
Thank you so much. fixed in 4ddbd58b40230b584a8c1a39ad987e83e2562ae2

**Comment by ghost** [comment]
you’re welcome :-)
